test-suite/certificates/client_private.key

```diff
@@ -0,0 +1,52 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
What are these certificates for?

Reply:
These are the very basic certificates that myself and Rio used when testing the harness, thought it might be a good idea to include the examples.